lib/unhangout-sockets.js

```diff
@@ -167,6 +167,17 @@ _.extend(UnhangoutSocketManager.prototype, events.EventEmitter.prototype, {
             }
         }, this));
 
+        mgr.on("store-contact", _.bind(function(socket, args) {
+            var event = this.getEvent(args.roomId);
+
+            var user = socket.user; 
+            user.set("preferredContact", args.preferredContact);
+            user.set("noShare", args.noShareVal);
```

Comment:
If we're making contact-info per-event, this needs to be changed to index by event.